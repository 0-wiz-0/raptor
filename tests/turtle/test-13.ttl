# Test for : allowed
@prefix :    <http://example.org/foo> .

[] : [] .

: : : .

