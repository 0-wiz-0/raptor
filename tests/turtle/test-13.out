_:genid1 <http://example.org/me#> _:genid2 .
<http://example.org/me#> <http://example.org/me#> <http://example.org/me#> .
